src/stories/components/ChecSlideoutPanel.stories.mdx

```diff
@@ -0,0 +1,377 @@
+                @close="openThird = false"
+              >
+                Hello, I'm the third panel!
+                <ChecSlideoutPanel
+                  v-if="openFourth"
+                  title="Limbo panel"
+                  size="3/4"
+                  overlay="light"
+                  titleTag="h2"
+                  width="2xl"
+                  closeOnOverlayClick
+                  @close="openFourth = false"
+                >
+                Hello, I'm the limbo inception layer!
+                <template v-slot:toolbar>
+                  <ChecButton @click="openFourth = false" color="primary" text-only class="p-0">
+                    Cancel
+                  </ChecButton>
+                </template>
+              </ChecSlideoutPanel>
+                <template v-slot:toolbar>
+                  <ChecButton
+                    variant="small"
+                    outline
+                    color="primary"
+                    @click="openFourth = true"
+                  >
+                    Open the fourth panel
+                  </ChecButton>
+                  <div class="flex">
+                    <ChecButton @click="openThird = false" color="primary" text-only class="p-0">
+                      Cancel
+                    </ChecButton>
+                    <ChecButton class="ml-5" color="primary">
+                      Save
+                    </ChecButton>
+                  </div>
+                </template>
+              </ChecSlideoutPanel>
+              <template v-slot:toolbar>
+                <ChecButton
+                  variant="small"
+                  outline
+                  color="primary"
+                  @click="openThird = true"
+                >
+                  Open the third panel
+                </ChecButton>
+                <div class="flex">
+                  <ChecButton @click="openSecond = false" color="primary" text-only class="p-0">
+                    Cancel
+                  </ChecButton>
+                  <ChecButton class="ml-5" color="primary">
+                    Save
+                  </ChecButton>
+                </div>
+              </template>
+            </ChecSlideoutPanel>
+          <template v-slot:toolbar>
+            <ChecButton
+              variant="small"
+              color="primary"
+              outline
+              @click="openSecond = true"
+            >
+              Open the second panel
+            </ChecButton>
+            <div class="flex">
+              <ChecButton @click="close" color="primary" text-only class="p-0">
+                Cancel
+              </ChecButton>
+              <ChecButton class="ml-5" color="primary">
+                Save
+              </ChecButton>
+            </div>
+          </template>
+        </ChecSlideoutPanel>
+        <ChecButton
+          @click="open = !open"
+          color="primary"
+          class="m-20"
+        >
+          Open the first panel
```

Comment:
After I click this, the button is still focussed. We should switch the focus when panels are opened away from whatever opened it. I can click this button then press enter repeatedly to toggle the panel's visibility. I think what should happen is that I press enter once and it opens up, then pressing enter again should trigger the "Save" button in the panel (assuming that the panel contains a form, and that the "Save" button is a submit button). Anyway - worth considering this.